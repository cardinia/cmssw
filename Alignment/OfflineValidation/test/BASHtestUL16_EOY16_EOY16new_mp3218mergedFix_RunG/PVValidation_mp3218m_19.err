27-Sep-2019 21:43:39 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/4C0245C1-1212-E911-8E8A-0CC47A7C3636.root
27-Sep-2019 21:43:42 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/4C0245C1-1212-E911-8E8A-0CC47A7C3636.root
Begin processing the 1st record. Run 279116, Event 814514580, LumiSection 451 on stream 0 at 27-Sep-2019 21:43:48.235 CEST
Begin processing the 1001st record. Run 279116, Event 822707675, LumiSection 455 on stream 0 at 27-Sep-2019 21:47:34.217 CEST
Begin processing the 2001st record. Run 279116, Event 829982557, LumiSection 459 on stream 0 at 27-Sep-2019 21:51:08.353 CEST
Begin processing the 3001st record. Run 279116, Event 838351287, LumiSection 463 on stream 0 at 27-Sep-2019 21:54:34.452 CEST
Begin processing the 4001st record. Run 279116, Event 846243673, LumiSection 468 on stream 0 at 27-Sep-2019 21:57:52.922 CEST
Begin processing the 5001st record. Run 279116, Event 855042212, LumiSection 472 on stream 0 at 27-Sep-2019 22:01:01.356 CEST
Begin processing the 6001st record. Run 279116, Event 862809526, LumiSection 476 on stream 0 at 27-Sep-2019 22:04:02.533 CEST
Begin processing the 7001st record. Run 279116, Event 869081095, LumiSection 480 on stream 0 at 27-Sep-2019 22:07:06.140 CEST
Begin processing the 8001st record. Run 279116, Event 877309306, LumiSection 484 on stream 0 at 27-Sep-2019 22:10:11.321 CEST
Begin processing the 9001st record. Run 280002, Event 6114234, LumiSection 9 on stream 0 at 27-Sep-2019 22:11:02.601 CEST
Begin processing the 10001st record. Run 280002, Event 16215792, LumiSection 22 on stream 0 at 27-Sep-2019 22:11:04.504 CEST
Begin processing the 11001st record. Run 280002, Event 25685643, LumiSection 35 on stream 0 at 27-Sep-2019 22:11:06.281 CEST
Begin processing the 12001st record. Run 280002, Event 34440430, LumiSection 47 on stream 0 at 27-Sep-2019 22:11:08.391 CEST
Begin processing the 13001st record. Run 280002, Event 43620726, LumiSection 60 on stream 0 at 27-Sep-2019 22:11:10.274 CEST
Begin processing the 14001st record. Run 280002, Event 53743108, LumiSection 73 on stream 0 at 27-Sep-2019 22:11:12.153 CEST
Begin processing the 15001st record. Run 280002, Event 62889500, LumiSection 86 on stream 0 at 27-Sep-2019 22:11:13.940 CEST
Begin processing the 16001st record. Run 280002, Event 71695719, LumiSection 99 on stream 0 at 27-Sep-2019 22:11:15.857 CEST
Begin processing the 17001st record. Run 280249, Event 2442985306, LumiSection 1382 on stream 0 at 27-Sep-2019 22:11:17.786 CEST
Begin processing the 18001st record. Run 280249, Event 2451416117, LumiSection 1386 on stream 0 at 27-Sep-2019 22:11:18.906 CEST
Begin processing the 19001st record. Run 280249, Event 2460818531, LumiSection 1391 on stream 0 at 27-Sep-2019 22:11:20.091 CEST
Begin processing the 20001st record. Run 280249, Event 2467070605, LumiSection 1395 on stream 0 at 27-Sep-2019 22:11:21.253 CEST
Begin processing the 21001st record. Run 280249, Event 2475865306, LumiSection 1399 on stream 0 at 27-Sep-2019 22:11:22.362 CEST
Begin processing the 22001st record. Run 280249, Event 2482275176, LumiSection 1403 on stream 0 at 27-Sep-2019 22:11:23.674 CEST
Begin processing the 23001st record. Run 280249, Event 2490540838, LumiSection 1407 on stream 0 at 27-Sep-2019 22:11:25.231 CEST
Begin processing the 24001st record. Run 280249, Event 2498069080, LumiSection 1411 on stream 0 at 27-Sep-2019 22:11:26.445 CEST
Begin processing the 25001st record. Run 280249, Event 2505934571, LumiSection 1415 on stream 0 at 27-Sep-2019 22:11:27.650 CEST
Begin processing the 26001st record. Run 280249, Event 2516148636, LumiSection 1420 on stream 0 at 27-Sep-2019 22:11:28.842 CEST
Begin processing the 27001st record. Run 280249, Event 2523717511, LumiSection 1424 on stream 0 at 27-Sep-2019 22:11:30.037 CEST
Begin processing the 28001st record. Run 280249, Event 2531578238, LumiSection 1428 on stream 0 at 27-Sep-2019 22:11:31.177 CEST
Begin processing the 29001st record. Run 280249, Event 2537407679, LumiSection 1432 on stream 0 at 27-Sep-2019 22:11:32.325 CEST
27-Sep-2019 22:11:32 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/4C0245C1-1212-E911-8E8A-0CC47A7C3636.root
27-Sep-2019 22:11:32 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/80BCDB03-3212-E911-BF0D-0CC47A78A360.root
27-Sep-2019 22:11:34 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/80BCDB03-3212-E911-BF0D-0CC47A78A360.root
Begin processing the 30001st record. Run 279116, Event 61187174, LumiSection 41 on stream 0 at 27-Sep-2019 22:14:38.670 CEST
Begin processing the 31001st record. Run 279116, Event 70463673, LumiSection 46 on stream 0 at 27-Sep-2019 22:18:02.778 CEST
Begin processing the 32001st record. Run 279116, Event 77275795, LumiSection 50 on stream 0 at 27-Sep-2019 22:22:02.687 CEST
Begin processing the 33001st record. Run 279116, Event 85198565, LumiSection 55 on stream 0 at 27-Sep-2019 22:26:05.595 CEST
Begin processing the 34001st record. Run 279116, Event 94402257, LumiSection 60 on stream 0 at 27-Sep-2019 22:29:53.447 CEST
Begin processing the 35001st record. Run 279116, Event 101362472, LumiSection 64 on stream 0 at 27-Sep-2019 22:33:21.605 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:35:07 CEST  Run: 279116 Event: 105935244
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 36001st record. Run 279116, Event 109848190, LumiSection 68 on stream 0 at 27-Sep-2019 22:36:45.439 CEST
Begin processing the 37001st record. Run 279116, Event 118020667, LumiSection 73 on stream 0 at 27-Sep-2019 22:40:05.461 CEST
Begin processing the 38001st record. Run 279116, Event 127060302, LumiSection 78 on stream 0 at 27-Sep-2019 22:43:30.935 CEST
Begin processing the 39001st record. Run 279116, Event 133863835, LumiSection 82 on stream 0 at 27-Sep-2019 22:46:49.893 CEST
Begin processing the 40001st record. Run 279116, Event 142959730, LumiSection 87 on stream 0 at 27-Sep-2019 22:50:16.197 CEST
Begin processing the 41001st record. Run 279116, Event 151276918, LumiSection 92 on stream 0 at 27-Sep-2019 22:53:38.468 CEST
Begin processing the 42001st record. Run 279116, Event 159365801, LumiSection 96 on stream 0 at 27-Sep-2019 22:57:03.220 CEST
Begin processing the 43001st record. Run 279116, Event 168299804, LumiSection 101 on stream 0 at 27-Sep-2019 23:00:30.805 CEST
Begin processing the 44001st record. Run 279116, Event 175859509, LumiSection 106 on stream 0 at 27-Sep-2019 23:03:50.732 CEST
Begin processing the 45001st record. Run 279116, Event 183985880, LumiSection 111 on stream 0 at 27-Sep-2019 23:07:19.310 CEST
Begin processing the 46001st record. Run 279116, Event 193266950, LumiSection 116 on stream 0 at 27-Sep-2019 23:10:43.835 CEST
Begin processing the 47001st record. Run 279116, Event 200056384, LumiSection 120 on stream 0 at 27-Sep-2019 23:14:08.735 CEST
Begin processing the 48001st record. Run 279116, Event 209259506, LumiSection 125 on stream 0 at 27-Sep-2019 23:17:31.155 CEST
Begin processing the 49001st record. Run 279116, Event 217685888, LumiSection 130 on stream 0 at 27-Sep-2019 23:20:57.537 CEST
27-Sep-2019 23:24:26 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016G/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/00000/80BCDB03-3212-E911-BF0D-0CC47A78A360.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       1        1
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          279116/105935244                  
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 1                   1
System                  6                   6

dropped waiting message count 0
[9.536MB/9.536MB][100%][==================================================][9.536MB/s]  [9.536MB/9.536MB][100%][==================================================][9.536MB/s]  
