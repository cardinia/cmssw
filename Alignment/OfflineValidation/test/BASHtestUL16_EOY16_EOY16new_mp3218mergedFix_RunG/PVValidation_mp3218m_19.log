000 (3196554.019.000) 09/27 21:33:44 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196554.019.000) 09/27 21:43:27 Started transferring input files
	Transferring to host: <188.184.142.249:9618?addrs=188.184.142.249-9618+[2001-1458-301-46--100-ee]-9618&noUDP&sock=4435_9bcd_13472>
...
040 (3196554.019.000) 09/27 21:43:27 Finished transferring input files
...
001 (3196554.019.000) 09/27 21:43:27 Job executing on host: <188.184.142.249:9618?addrs=188.184.142.249-9618+[2001-1458-301-46--100-ee]-9618&noUDP&sock=2414_61b6_3>
...
006 (3196554.019.000) 09/27 21:43:36 Image size of job updated: 222828
	117  -  MemoryUsage of job (MB)
	119236  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 21:48:37 Image size of job updated: 560296
	545  -  MemoryUsage of job (MB)
	557292  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 21:53:38 Image size of job updated: 564188
	549  -  MemoryUsage of job (MB)
	561184  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 22:13:40 Image size of job updated: 595272
	579  -  MemoryUsage of job (MB)
	592264  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 22:18:40 Image size of job updated: 596020
	580  -  MemoryUsage of job (MB)
	593012  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 22:23:40 Image size of job updated: 597484
	581  -  MemoryUsage of job (MB)
	594476  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 22:38:42 Image size of job updated: 598592
	582  -  MemoryUsage of job (MB)
	595584  -  ResidentSetSize of job (KB)
...
006 (3196554.019.000) 09/27 22:43:42 Image size of job updated: 600060
	584  -  MemoryUsage of job (MB)
	597052  -  ResidentSetSize of job (KB)
...
040 (3196554.019.000) 09/27 23:24:44 Started transferring output files
...
040 (3196554.019.000) 09/27 23:24:44 Finished transferring output files
...
005 (3196554.019.000) 09/27 23:24:44 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:39:14, Sys 0 00:00:10  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:39:14, Sys 0 00:00:10  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	9989  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	9989  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9813        1   2015222 
	   Memory (MB)          :      584     2000      2000 
...
