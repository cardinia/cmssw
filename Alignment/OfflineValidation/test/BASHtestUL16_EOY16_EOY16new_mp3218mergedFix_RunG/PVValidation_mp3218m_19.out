Job started at  Fri Sep 27 21:43:27 CEST 2019
batch dir: /pool/condor/dir_30556 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_279116_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.6M -rw-r--r--. 1 tkello zh 9.6M Sep 27 23:24 PVValidation_mp3218m_279116.root -rw-r--r--. 1 tkello zh 21K Sep 27 21:43 PVValidation_mp3218m_279116_cfg.py -rw-r--r--. 1 tkello zh 8.6K Sep 27 23:24 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:43 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:43 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:43 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:43 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:43 var
Job ended at  Fri Sep 27 23:24:44 CEST 2019
